src/services/MQTTRoomService/MQTTRoomService.ts

```
@@ -0,0 +1,115 @@
+          eventData &&
+          eventData.type === "Buffer" &&
+          Array.isArray(eventData.data)
+        ) {
+          eventData = Buffer.from(eventData.data);
+        }
+
+        room.emit(eventName, eventData);
+      }
+    }
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)